webapp/handlers.py

```diff
@@ -52,10 +52,18 @@ def page_not_found(error):
 
         return flask.render_template("404.html", error=error.description), 404
 
+    @app.errorhandler(500)
```

Comment:
```suggestion
    @app.errorhandler(502)
    def internal_error(error):
        return flask.render_template("50X.html", error=error), 502
```

Reply:
we don't return 500 errors, only 502 `Bad Gateway`

Reply:
Even if there is internal server error (typo in backend code, etc)?

Reply:
@tbille It is possible to get 500 (if something is messed in the code):

<img width="1875" alt="screen shot 2018-10-18 at 16 10 32" src="https://user-images.githubusercontent.com/83575/47160544-744d3100-d2f0-11e8-8be8-04fe62b75778.png">

Sure, we will catch most of there during development or review/QA, but it is possible that some code would fail internally on prod.

Reply:
Can the same handler be used for multiple codes?

```python
@app.errorhandler(500)
@app.errorhandler(501)
@app.errorhandler(502)
def error_handler(error):
   return flask.render_template("50X.html", error=error), error.code
```

Reply:
Yes you can, and please do. You should definitely catch `500` as well as `502` and any [other `5xx` error](https://en.wikipedia.org/wiki/List_of_HTTP_status_codes#5xx_Server_errors) that might possibly happen. I would suggest you at least catch `500` to `505`.